python/dgl/distributed/rpc.py

```diff
@@ -777,6 +748,11 @@ def remote_call_to_machine(target_and_requests, timeout=0):
         if res_cls is not None:
             num_res += 1
             msgseq2pos[msg_seq] = pos
+    return num_res, msgseq2pos
+
+def recv_responses(num_res, msgseq2pos, timeout=0):
+    myrank = get_rank()
+    all_res = [None] * num_res
```

Comment:
is this right? @aksnzhy

Reply:
I think here is no problem.

Reply:
Can we re-use `recv_response` and `send_request` instead of re-implement all the logic in `recv_responses` and `send_requests`?

Reply:
i didn't reimplement these. the code of `recv_responses` and `send_requests` are copied from `remote_call_to_machine`. if we could reuse `recv_response` and `send_request`, we could reuse them in `remote_call_to_machine`.